src/ComputeSharp.D2D1/Shaders/Interop/D2D1ResourceTextureManager.cs

```diff
@@ -0,0 +1,487 @@
+/// </summary>
+/// <remarks>
+/// <para>
+/// The built-in <see href="https://docs.microsoft.com/en-us/windows/win32/api/d2d1_1/nn-d2d1_1-id2d1effect"><c>ID2D1Effect</c></see>
+/// implementation provided by <see cref="D2D1PixelShaderEffect"/> uses resource texture manager objects to handle resource textures.
+/// </para>
+/// <para>
+/// These managers are COM objects implementing the following interface:
+/// <code>
+/// [uuid(3C4FC7E4-A419-46CA-B5F6-66EB4FF18D64)]
+/// interface ID2D1ResourceTextureManager : IUnknown
+/// {
+///     HRESULT Initialize(
+///         [in, optional] const GUID                             *resourceId,
+///         [in]           const D2D1_RESOURCE_TEXTURE_PROPERTIES *resourceTextureProperties,
+///         [in, optional] const BYTE                             *data,
+///         [in, optional] const UINT32                           *strides,
+///                        UINT32                                 dataSize);
+/// 
+///     HRESULT Update(
+///         [in, optional] const UINT32 *minimumExtents,
+///         [in, optional] const UINT32 *maximimumExtents,
+///         [in]           const UINT32 *strides,
+///                        UINT32       dimensions,
+///         [in]           const BYTE   *data,
+///                        UINT32       dataCount);
+/// };
+/// </code>
+/// </para>
+/// <para>
+/// These can be used to create resource textures, pass them to effects, and also share resources among different effects.
+/// </para>
+/// <para>
+/// For details of how these work, <c>Initialize</c> maps to <c>ID2D1EffectContext::CreateResourceTexture</c>, whereas
+/// <c>Update</c> maps to <c>ID2D1ResourceTexture::Update</c>. The same behavior and parameters as those methods is used.
+/// </para>
+/// <para>
+/// This interface is implemented by ComputeSharp.D2D1, and it can be used through the APIs in <see cref="D2D1ResourceTextureManager"/>.
+/// That is, <see cref="Create"/> is first use to create an <c>ID2D1ResourceTextureManager</c> instance. Then, <see cref="Initialize"/>
+/// can be used to initialize the resource texture held by the manager. The manager can also be assigned to an effect at any time,
+/// using the available property indices from <see cref="D2D1PixelShaderEffectProperty"/>. To update texture data after its initial
+/// creation, <see cref="Update(void*, ReadOnlySpan{uint}, ReadOnlySpan{uint}, ReadOnlySpan{uint}, ReadOnlySpan{byte})"/> can be used.
+/// </para>
+/// <para>
+/// An RCW (runtime callable wrapper, see <see href="https://docs.microsoft.com/dotnet/standard/native-interop/runtime-callable-wrapper"/>),
+/// is also available for all of these APIs, implemented by the same <see cref="D2D1PixelShaderEffect"/>. In this case, the constructor can
+/// be used to create and initialize an instance (equivalent to calling <see cref="Create"/> and <see cref="Initialize"/>, and then
```

Comment:
proofreading nit: missing a closing parenthesis after `<see cref="Initialize"/>`

Reply:
Fixed in 28ed19c349592ecaafaafd00328df254e7556db9.